conanfile.py

```diff
@@ -23,18 +23,19 @@ class VehicleAppCppSdkConan(ConanFile):
     license = "Apache-2.0"
     url = "https://github.com/eclipse-velocitas/vehicle-app-cpp-sdk"
     description = "The Vehicle App SDK for c++ allows to create Vehicle Apps from the Velocitas development model in the c++ programming language."
-    # In general: Pin recipe revisions of depenencies having further dependencies to avoid build issues due to updated recipes
+    # In general: Pin recipe revisions of dependencies having further dependencies to avoid build issues due to updated recipes
     # Workaround1: Pin recipe revision for transient dependency googleapis for enabling the container build
     # Workaround2: Pin recipe revision for transient dependency paho-mqtt-c cause latest is pulling libanl which cannot be found
     requires = \
+        "c-ares/1.19.1@#420a0b77e370f4b96bee88ef91837ccc", \
         "cpr/1.9.3@#26c6732664548385b583b3a0116b8d5e", \
```

Comment:
Maybe cpr could be also increased to "latest greatest"

Reply:
Done